docs/documentation/features/icons.md

```diff
@@ -12,7 +12,7 @@ Bundle: `org.eclipse.smarthome.ui.icon`
 
 Eclipse SmartHome comes with a flexible infrastructure for handling icons that are to be used within user interfaces.
 This bunde registers a servlet under the url `/icon/`, which can then be easily queried for icons using GET requests of the form `/icon/<category>?state=<state>&format=[png|svg]&iconset=<iconsetid>`, where 
-- `category` is one from the [list of channel categories](../../development/bindings/thing-definition.html#channel-categories) or any custom category that might be used within the solution
+- `category` is one from the [list of channel categories](../../development/bindings/categories.html#channel-categories) or any custom category that might be used within the solution
```

Comment:
Looking at why the link is broken today already, I would guess the correct link would be `../development/bindings/categories.html#channel-categories`.